docs/source/samples/21_mobilenet_decoding_on_device.rst

```diff
@@ -1,32 +1 @@
-21 - RGB & MobilenetSSD decoding on device
-==========================================
-
-This example shows how to run MobileNetv2SSD on the RGB input frame, and how to display both the RGB
-preview and the metadata results from the MobileNetv2SSD on the preview. It's similar to example '08_rgb_mobilenet' except
-decoding is done on Myriad instead on the host.
-
-
-setConfidenceThreshold - confidence threshold above which objects are detected
-
-Demo
-####
-
-
-Setup
-#####
-
-.. include::  /includes/install_from_pypi.rst
-
-This example also requires MobilenetSDD blob (:code:`mobilenet.blob` file) to work - you can download it from
-`here <https://artifacts.luxonis.com/artifactory/luxonis-depthai-data-local/network/mobilenet-ssd_openvino_2021.2_6shave.blob>`__
-
-Source code
-###########
-
-Also `available on GitHub <https://github.com/luxonis/depthai-python/blob/main/examples/21_mobilenet_device_side_decoding.py>`__
-
-.. literalinclude:: ../../../examples/21_mobilenet_device_side_decoding.py
-   :language: python
-   :linenos:
-
-.. include::  /includes/footer-short.rst
+This example was moved to :ref:`08 - RGB & MobilenetSSD`
```

Comment:
If we moved it to demo 08, should we also remove the url to this page in the index.rst?

Reply:
it didn't show up in the index toc, but removed it anyway, thanks!
![image](https://user-images.githubusercontent.com/5244214/113627963-81973b00-9664-11eb-84b8-3a5ab8ed7a6e.png)